BookPlayer/Models/Playlist+CoreDataClass.swift

```diff
@@ -164,6 +163,13 @@ public class Playlist: LibraryItem {
 
         return "\(count) Files"
     }
+
+    public override func setCompletionState(isComplete: Bool = true) {
+        books?.forEach { book in
+            guard let book = book as? Book else { return }
+            book.setCompletionState()
```

Comment:
this should receive the parameter `isComplete` that is being passed in the function, currently if I set a playlist to not complete, it will still mark the books as completed